backend/api.py

```diff
@@ -1,9 +1,22 @@
 from fastapi import FastAPI
 from fastapi.responses import JSONResponse
 from director import question
+from fastapi.middleware.cors import CORSMiddleware
 
 app = FastAPI()
 
+origins = [
+    "http://localhost:8650",
```

Comment:
Fine for now but ideally we'd want this to be configurable.

Reply:
Will amend on a new PR